modules/installation-configuring-shiftstack-single-ipv6.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// TBD
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-shiftstack-single-ipv6_{context}"]
+= Configuring a cluster with single-stack IPv6 networking
+
+You can create a single-stack IPv6 cluster on {rh-openstack-first}.
+
+.Prerequisites
+
+* Your {rh-openstack} deployment has an existing IPv6 subnet.
+* DNS is configured for the existing IPv6 subnet.
+* The IPv6 subnet is added to a networking service (Neutron) router, and the router is configured to send router advertisements (RAs). 
+* The {rh-openstack} network accepts a minimum MTU size of 1380 bytes.
+* You created API and ingress virtual IP addresses (VIPs) as ports on the network and included those addresses in the `install-config.yaml` file.
+
+.Procedure
+
+== Creating Network for the Cluster
+
+. Create the network and IPv6 subnet:
+  [source,bash]
+  ----
+  $ openstack network create --project <project-name> --share --external --provider-physical-network <physical-network> --provider-network-type flat v6-network
+  ----
+
+  [source,bash]
+  ----
+  $ openstack subnet create --project <project-name> v6-subnet --subnet-range fd2e:6f44:5dd8:c956::/64 --dhcp --dns-nameserver <dns-address> --network v6-network --ip-version 6 --ipv6-ra-mode stateful --ipv6-address-mode stateful
+  ----
+
+. Add the network to the router's external gateway to enable external connectivity and router advertisements:
+  [source,bash]
+  ----
+  $ openstack router set --external-gateway v6-network <router-id>
```

Comment:
can also add the following note:

https://github.com/openshift/installer/commit/5fd62ea137f9782601bac9d551cfbd4aa441cd9f

Reply:
How 'bout:

> NOTE: You must add any additional IPv6 subnets that are used in the cluster to an {rh-openstack} router to facilitate router advertisements.

Reply:
https://github.com/openshift/openshift-docs/pull/85554/files#r1936400925